great_expectations/expectations/registry.py

```diff
@@ -307,16 +307,30 @@ def register_metric(  # noqa: PLR0913 # FIXME CoP
     return res
 
 
-def get_metric_provider(
-    metric_name: str, execution_engine: ExecutionEngine
+def _get_metric_definition(metric_name: str) -> dict:
+    try:
+        return _registered_metrics[metric_name]
+    except KeyError:
+        raise gx_exceptions.MetricProviderError(f"No definition found for {metric_name}")  # noqa: TRY003 # FIXME CoP
+
+
+def get_sqlalchemy_metric_provider(
```

Comment:
I added this helper method because I didn't want to reach into the _registered_metrics from tests. I also didn't want to need a real sqlalchemy execution engine in tests because we don't have sqlalchemy connectors in unit tests and I didn't want to add that dependency. This felt ok because this is the default providers.